srcpkgs/audiotube/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'audiotube'
+pkgname=audiotube
+version=22.04
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_PREFIX_PATH=/usr/lib/python3.10/site-packages/pybind11/share/cmake"
+hostmakedepends="extra-cmake-modules qt5-host-tools qt5-qmake"
+makedepends="qt5-devel kirigami2-devel ki18n-devel kcoreaddons-devel kcrash-devel gettext-devel python3-pybind11 yt-dlp python3-ytmusicapi"
```

Comment:
wrap at 80 columns

Reply:
I'll fix it. Do you have any ideas why CMake isn't finding binaries in some specific arches?